components/com_content/src/View/Category/FeedView.php

```diff
@@ -51,10 +51,17 @@ protected function reconcileNames($item)
         $app               = Factory::getApplication();
         $params            = $app->getParams();
         $item->description = '';
-        $obj               = json_decode($item->images);
+        $obj               = json_decode($item->images) ?? new \stdClass();
 
+        // Ensure alt properties are set
+        $obj->image_intro_alt    = $obj->image_intro_alt ?? '';
+        $obj->image_fulltext_alt = $obj->image_fulltext_alt ?? '';
+
+        // Set feed image to image_intro or if that's empty, to image_fulltext
         if (!empty($obj->image_intro)) {
             $item->description = '<p>' . HTMLHelper::_('image', $obj->image_intro, $obj->image_intro_alt) . '</p>';
+        } elseif (!empty($obj->image_fulltext)) {
+            $item->description = '<p>' . HTMLHelper::_('image', $obj->image_fulltext, $obj->image_fulltext_alt) . '</p>';
```

Comment:
remove you check/set and mv it inside the if
```suggestion
        // Set feed image to image_intro or if that's empty, to image_fulltext
        if (!empty($obj->image_intro)) {
            $item->description = '<p>' . HTMLHelper::_('image', $obj->image_intro, $obj->image_intro_alt ?? '') . '</p>';
        } elseif (!empty($obj->image_fulltext)) {
            $item->description = '<p>' . HTMLHelper::_('image', $obj->image_fulltext, $obj->image_fulltext_alt ?? '') . '</p>';
```